src/components/appFolder/App.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import styled from 'styled-components';
+
+import Dropdown from 'components/dropdownFolder/Dropdown'
+import Options from 'components/optionsFolder/Options'
+import TextField from 'components/textfieldFolder/TextField'
+import Button from 'components/buttonFolder/Button'
+import Breadcrumb from 'components/breadcrumbFolder/Breadcrumb'
+import Tabs from 'components/tabFolder/Tab'
+import Pane from 'components/tabFolder/Pane'
+
+const Wrapper = styled.div`
+  margin-left: 50px;
+  margin-top: 20px;
+  flex-direction: column;
+  alignItems: 'center'
+`;
+
+const Section2 = styled.div`
+  
+`;
+
+const HeadingText = styled.div`
+  font-size: 1.9em;
+  color: #a8a8a8;
+  margin-bottom: 20px;
+  font-family: Lato-Black;
```

Comment:
Try not to redeclare things like `font-family`. CSS properties "cascade" so you can set properties like `font-face` at the top of the tree (eg `html` tag) and it will be inherited by all it's children.

You can then use `font-weight` for bold, etc.